courses/free-courses-ne.md

```diff
@@ -43,3 +45,7 @@
 
 * [WordPress Complete Tutorial In Nepali](https://www.youtube.com/playlist?list=PL2OJkQtHPRie2xyBApANdVp_LUz4v7xIG) - Technology Channel
 
+
+#### PHP
+
+* [PHP Basic Tutorial In Nepali](https://youtu.be/Ec8bwfCNhkA?si=WgnbspeofHlg65VE) - Technology Channel
```

Comment:
Missing newline character at end of file

Reply:
its done now